src/components/tree-detail/last-waterings/last-waterings.tsx

```diff
@@ -71,12 +71,20 @@ export const LastWaterings: React.FC<LastWateringsProps> = ({
 					<WateringSection
 						waterings={wateringsThisMonth}
 						title={i18n.treeDetail.lastWaterings.thisMonth}
-						noWateringsHint={i18n.treeDetail.lastWaterings.nothingThisMonth}
+						noWateringsHint={
+							wateringsThisWeek.length > 0
+								? i18n.treeDetail.lastWaterings.nothingMoreThisMonth
+								: i18n.treeDetail.lastWaterings.nothingThisMonth
+						}
 					/>
 					<WateringSection
 						waterings={wateringsThisYear}
 						title={i18n.treeDetail.lastWaterings.thisYear}
-						noWateringsHint={i18n.treeDetail.lastWaterings.nothingThisYear}
+						noWateringsHint={
+							wateringsThisWeek.length || wateringsThisMonth.length > 0
```

Comment:
this is not totally correct, we need `wateringsThisWeek.length > 0 || wateringsThisMonth.length > 0`. However, the inline condition (in combination with the ternary operator) starts to get long and difficult to read and it might be worth it to extract the condition in a named constant that is declared before the return block, e.g. `const hasRecentWaterings = ...`. (not sure how to name it though)